### ISSUE TITLE
PackageDev build system and ST2 compatibility

### ISSUE THREAD

**Issue by Phidica** [toru-hamaguchi/sublime-fish-shell#12]
The current project is written in a `.JSON-tmLanguage` file, which PackageDev parses to make a `.tmLanguage` file compatible with Sublime Text 2 or 3. That was a while ago, and PackageDev now recommends using a `.YAML-tmLanguage` file instead, and provides a command to convert `.JSON-tmLanguage` into `.YAML-tmLanguage`. I vote to make this switch 

~~Secondary to this is the fact that ST3 supports a `.sublime-syntax` format as well, which is pure YAML and a fair bit cleaner than `.YAML-tmLanguage` (albeit lacking in RegEx syntax highlighting), plus doesn't need to go through an extra parsing process. Only problem is the lack of backwards compatibility to ST2. If there exists a converter to turn `.sublime-syntax` into `.tmLanguage`, then I would prefer to just move all development into a `.sublime-syntax` file and let the converter make the `.tmLanguage` file for ST2 users~~

tl;dr, we should move development into ~~either~~ `.YAML-tmLanguage` ~~or `.sublime-syntax`~~

Edit: The advantages of `.YAML-tmLanguage` over `.sublime-syntax` are easy backwards compatibility and all the power that PackageDev brings. There are virtually no downsides. I've been using it on my fork of the repo and I'm sold. I can submit a pull request from [my dev branch](https://github.com/Rhotias/sublime-fish-shell/tree/PackageDev-YAML-development) if it's agreed to make this switch

**Comment by Phidica** [comment]
As this repo now appears to be inactive, I have converted my fork into a standalone alternative and will *not* be merging it into this repo. Check it out [here](https://github.com/Phidica/sublime-fish-shell-improved.git)
